lib/ServerDate.js

```diff
@@ -290,4 +291,4 @@ synchronize();
 
 // Return the newly defined module.
 return ServerDate;
-})
+})((new Date()).getTime());
```

Comment:
also done.

--------------------------------------
James C.
james.conley@gmail.com

On Mon, Sep 28, 2015 at 7:24 PM, David Braun <notifications@github.com>
wrote:

> In lib/ServerDate.js
> <https://github.com/NodeGuy/ServerDate/pull/25#discussion_r40618436>:
>
> > @@ -290,4 +291,4 @@ synchronize();
> >
> >  // Return the newly defined module.
> >  return ServerDate;
> > -})
> > +})((new Date()).getTime());
>
> This can be simplified to Date.now().
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NodeGuy/ServerDate/pull/25/files#r40618436>.
>